packages/x-date-pickers-pro/src/DesktopDateRangePicker/DesktopDateRangePicker.tsx

```diff
@@ -6,8 +6,14 @@ import {
   usePickerState,
   DateInputPropsLike,
   DesktopWrapperProps,
+  DesktopWrapperSlotsComponent,
+  DesktopWrapperSlotsComponentProps,
```

Comment:
Should be named `DesktopWrapperSlotsComponentsProps`

x other variables named ` XXXComponentProps`